visual-testing-app/index.html

```diff
@@ -6,18 +6,10 @@
       name="viewport"
       content="width=device-width, initial-scale=1, viewport-fit=cover"
     />
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=Open+Sans:ital,wght@0,300;0,400;0,700;1,400;1,700&display=swap" rel="stylesheet">
     <title>UI-Kit | Visual testing app</title>
```

Comment:
There were 2 fonts loaded. One used by ui-kit, one used by the testing app. I felt it's unnecessary to load a custom font, just to use for the title of the violet test-case-header so I removed it as well.